Library/Homebrew/cask/installer.rb

```diff
@@ -156,7 +156,9 @@ def install
 
       oh1 "Installing Cask #{Formatter.identifier(@cask)}"
       # GitHub Actions globally disables Gatekeeper.
-      opoo_outside_github_actions "macOS's Gatekeeper has been disabled for this Cask" unless quarantine?
+      opoo_outside_github_actions(
+        "You are overriding your security settings to install an app from an unidentified developer."
+      ) unless quarantine?
```

Comment:
```suggestion
      unless quarantine?
        opoo_outside_github_actions "--no-quarantine bypasses macOS’s Gatekeeper, reducing system security. " \
                                    "Do not use this flag unless you understand the risks."
      end
```

Reply:
I think the reference to the flag is required context for the user to know why they are overriding the security. Otherwise it is a bit ambiguous. O